Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1,10 +1,9 @@
-import socket
-
 import demistomock as demisto
 # Common functions script
 # =======================
 # This script will be appended to each server script before being executed.
 # Please notice that to add custom common code, add it to the CommonServerUserPython script
+import socket
 from datetime import datetime, timedelta
```

Comment:
if you already touched this section lets make it according to PEP8, and maybe by length so it will be more readable

Reply:
can't. the common_server_docs.py script is failing with normal imports. 
I'm suggesting to let it be.